<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="TrackingWorkflowExceptionEventArgs.xml" source-language="en-US" target-language="zh-CN">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac507e533d202a4b266d49202c8122334812de6db87.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07e533d202a4b266d49202c8122334812de6db87</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>Contains data associated with an exception that occurs during the execution of a workflow instance.</source>
          <target state="translated">包含与工作流实例执行过程中发生的异常关联的数据。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>When an exception occurs during the execution of a workflow instance and a <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingProfile&gt;</ph> that is associated with the workflow instance includes a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackPoint&gt;</ph> configured for a <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent.Exception?displayProperty=nameWithType&gt;</ph>, the workflow tracking infrastructure sets <ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord.EventArgs%2A&gt;</ph> to an <ph id="ph5">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> in the <ph id="ph6">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph> that it sends to the tracking service.</source>
          <target state="translated">当工作流实例执行过程中发生异常，并且与该工作流实例关联的 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingProfile&gt;</ph> 包括针对 <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackPoint&gt;</ph> 配置的 <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowEvent.Exception?displayProperty=nameWithType&gt;</ph> 时，工作流跟踪基础结构会在它发送到跟踪服务的 <ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord.EventArgs%2A&gt;</ph> 中将 <ph id="ph5">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> 设置为 <ph id="ph6">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> is used only by the runtime tracking service to pass information in a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> 仅由运行时跟踪服务使用，以在 <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.WorkflowTrackingRecord&gt;</ph> 中传递信息。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>The following code example demonstrates a method, named <ph id="ph1">`WriteExceptionEventArgs`</ph>, which captures a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>.</source>
          <target state="translated">下面的代码示例演示一个名为 <ph id="ph1">`WriteExceptionEventArgs`</ph> 的方法，该方法可捕获 <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>The code checks whether the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">代码将检查 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性是否为 <ph id="ph2">`null`</ph>（在 Visual Basic 中为 <ph id="ph3">`Nothing`</ph>）。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>If it is not, the code writes the message associated with the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property to the console.</source>
          <target state="translated">如果不是，代码会将与 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性关联的消息写入控制台。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>In addition, the code converts the value of the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> property to a string and writes it to the console.</source>
          <target state="translated">此外，代码会将 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> 属性的值转换为字符串，并将其写入控制台。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>This code example is part of the EventArgs Tracking SDK sample from the Program.cs file.</source>
          <target state="translated">此代码示例摘自 Program.cs 文件中的 EventArgs 跟踪 SDK 示例。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs">
          <source>For more information, see <bpt id="p1">[</bpt>EventArgs Tracking Sample<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>.</source>
          <target state="translated">有关详细信息，请参阅<bpt id="p1">[</bpt>EventArgs 跟踪示例<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.ContextGuid">
          <source>Gets the context ID of the associated activity.</source>
          <target state="translated">获取关联的活动的上下文 ID。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.ContextGuid">
          <source>A <ph id="ph1">&lt;see cref="T:System.Guid" /&gt;</ph> that contains the context ID of the <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph>.</source>
          <target state="translated">一个 <ph id="ph1">&lt;see cref="T:System.Guid" /&gt;</ph>，其中包含 <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph> 的上下文 ID。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source>Gets the <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph> of the activity currently throwing the exception.</source>
          <target state="translated">获取当前引发异常的活动的 <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source>The <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph> of the activity currently throwing the exception.</source>
          <target state="translated">当前引发异常的活动的 <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source>Exceptions are passed up the chain of exception handler activities.</source>
          <target state="translated">异常将在一系列异常处理程序活动中向上传递。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source>An exception handler may choose to throw an exception again.</source>
          <target state="translated">异常处理程序可能会选择再次引发异常。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source>When an exception handler throws the exception again, a new exception tracking event containing a new <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> is raised to the runtime tracking infrastructure.</source>
          <target state="translated">当异常处理程序再次引发异常时，将会在运行时跟踪基础结构中引发一个包含新 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> 的新异常跟踪事件。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath">
          <source><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath%2A&gt;</ph> refers to the activity that is currently throwing the exception.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.CurrentActivityPath%2A&gt;</ph> 引用当前引发异常的活动。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>Gets the <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that is being thrown by the workflow instance.</source>
          <target state="translated">获取工作流实例所引发的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>The exception that is being thrown by the workflow instance.</source>
          <target state="translated">工作流实例引发的异常 。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>The following code example demonstrates a method, named <ph id="ph1">`WriteExceptionEventArgs`</ph>, which captures a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>.</source>
          <target state="translated">下面的代码示例演示一个名为 <ph id="ph1">`WriteExceptionEventArgs`</ph> 的方法，该方法可捕获 <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>The code checks whether the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">代码将检查 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性是否为 <ph id="ph2">`null`</ph>（在 Visual Basic 中为 <ph id="ph3">`Nothing`</ph>）。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>If it is not, the code writes the message associated with the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property to the console.</source>
          <target state="translated">如果不是，代码会将与 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性关联的消息写入控制台。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>If Exception is <ph id="ph1">`null`</ph> (<ph id="ph2">`Nothing`</ph>), the code converts the value of the <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> property to a string and writes it to the console.</source>
          <target state="translated">如果异常为 <ph id="ph1">`null`</ph> (<ph id="ph2">`Nothing`</ph>)，则代码会将 <ph id="ph3">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> 属性的值转换为字符串，并将其写入控制台。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>This code example is part of the EventArgs Tracking SDK Sample from the Program.cs file.</source>
          <target state="translated">此代码示例摘自 Program.cs 文件中的 EventArgs 跟踪 SDK 示例。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception">
          <source>For more information, see <bpt id="p1">[</bpt>EventArgs Tracking Sample<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>.</source>
          <target state="translated">有关详细信息，请参阅<bpt id="p1">[</bpt>EventArgs 跟踪示例<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>Gets the <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph> of the activity that originally threw the exception.</source>
          <target state="translated">获取原来引发异常的活动的 <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>The <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph> of the activity that originally threw the exception.</source>
          <target state="translated">原来引发异常的活动的 <ph id="ph1">&lt;see cref="P:System.Workflow.ComponentModel.Activity.QualifiedName" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>Exceptions are passed up the chain of exception handler activities.</source>
          <target state="translated">异常将在一系列异常处理程序活动中向上传递。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>An exception handler may choose to throw an exception again.</source>
          <target state="translated">异常处理程序可能会选择再次引发异常。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>When an exception handler throws an exception again, a new exception tracking event containing a new <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> is raised to the runtime tracking infrastructure.</source>
          <target state="translated">当异常处理程序再次引发异常时，将会在运行时跟踪基础结构中引发一个包含新 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph> 的新异常跟踪事件。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> refers to the activity that originally threw the exception.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> 引用原来引发异常的活动。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>The following code example demonstrates a method, named <ph id="ph1">`WriteExceptionEventArgs`</ph>, which captures a <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>.</source>
          <target state="translated">下面的代码示例演示一个名为 <ph id="ph1">`WriteExceptionEventArgs`</ph> 的方法，该方法可捕获 <ph id="ph2">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>The code checks whether the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph> in Visual Basic).</source>
          <target state="translated">代码将检查 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性是否为 <ph id="ph2">`null`</ph>（在 Visual Basic 中为 <ph id="ph3">`Nothing`</ph>）。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>If it is not, the code writes the message associated with the <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> property to the console.</source>
          <target state="translated">如果不是，代码会将与 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 属性关联的消息写入控制台。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>If <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> is <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>), the code converts the value of the <ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> property to a string and writes it to the console.</source>
          <target state="translated">如果 <ph id="ph1">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.Exception%2A&gt;</ph> 为 <ph id="ph2">`null`</ph> (<ph id="ph3">`Nothing`</ph>)，代码会将 <ph id="ph4">&lt;xref:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath%2A&gt;</ph> 属性的值转换为字符串，并将其写入控制台。</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>This code example is part of the EventArgs Tracking SDK Sample from the Program.cs file.</source>
          <target state="translated">此代码示例摘自 Program.cs 文件中的 EventArgs 跟踪 SDK 示例。</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.OriginalActivityPath">
          <source>For more information, see <bpt id="p1">[</bpt>EventArgs Tracking Sample<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>.</source>
          <target state="translated">有关详细信息，请参阅<bpt id="p1">[</bpt>EventArgs 跟踪示例<ept id="p1">](http://msdn.microsoft.com/library/94818da2-c3c4-4a13-b47a-a0fb30dbbf1c)</ept>。</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.ParentContextGuid">
          <source>Gets the context ID of the parent activity.</source>
          <target state="translated">获取父活动的上下文 ID。</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Workflow.Runtime.Tracking.TrackingWorkflowExceptionEventArgs.ParentContextGuid">
          <source>A <ph id="ph1">&lt;see cref="T:System.Guid" /&gt;</ph> that contains the context ID of the parent <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph>.</source>
          <target state="translated">一个 <ph id="ph1">&lt;see cref="T:System.Guid" /&gt;</ph>，其中包含父 <ph id="ph2">&lt;see cref="T:System.Workflow.ComponentModel.Activity" /&gt;</ph> 的上下文 ID。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>